src/applications/personalization/connected-accounts/components/AppDeletedAlert.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
+
+export class AppDeletedAlert extends React.Component {
+  componentDidMount() {
+    this.alert.focus();
+  }
+
+  alertMessage() {
+    // eslint-disable-next-line prettier/prettier
+    return `${this.props.account.attributes.title} has been disconnected from your account.`;
+  }
+
+  render() {
+    return (
+      <div
```

Comment:
I included this wrapper div so I can focus the <AlertBox /> without having to know anything about the structure of that component. I'm open to changing this!

Reply:
I think this is a good defensive coding strategy. 

I finally looked through the code and understand how we're setting a live region with declaring one outside the `<AlertBox />` element. [If the alert is not visible](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/3d15671aaef50637892a101842ef5d6d78a29485/packages/formation-react/src/components/AlertBox/AlertBox.jsx#L36), we're rendering a `<div aria-live="polite"></div>` by default.

I'll confirm correct behavior in Staging for JAWS and NVDA, but do not anticipate any issues.